### ISSUE TITLE
Fix Makefile error during compilation

### ISSUE THREAD

**Issue by jbenden** [jbenden/aircrack-ng-migration#1520]
**Reported by Jano on 2 Nov 2014 19:50 UTC**

Fix manpage/Makefile error

```
install: impossibile eseguire stat di "airdriver-ng.8": File o directory non esistente
make[*** [install](1]:) Errore 1
make[uscita dalla directory "/home/jano/aircrack-ng/manpages"
make: *** [install](1]:) Errore 2
```
- Patch attacked.

Jano 

Migrated-From: https://trac.aircrack-ng.org/ticket/1520

**Comment by jbenden** [comment]
**Attachment by Jano on 2 Nov 2014 19:52 UTC**

Uploaded file [makefile_airdriver-ng.patch](https://github.com/aircrack-ng/trac-attachments/blob/master/1520/20141102-195257_710-makefile_airdriver-ng.patch)

fix manpage/makefile error

**Comment by jbenden** [comment]
**Attachment by Jano on 2 Nov 2014 19:53 UTC**

Uploaded file [makefile_airdriver-ng.2.patch](https://github.com/aircrack-ng/trac-attachments/blob/master/1520/20141102-195322_578-makefile_airdriver-ng.2.patch)

fix manpage/makefile error

**Comment by jbenden** [comment]
**Comment by misterx on 2 Nov 2014 19:50 UTC**

Fixed in aircrack-ng/aircrack-ng@d3710a58e75cf4618f3f0efebee86798cc9ba88c. Thank you.
